openwisp_monitoring/device/api/serializers.py

```diff
@@ -20,6 +22,26 @@ class Meta:
         model = DeviceMonitoring
 
 
+class WifiSessionSerializer(serializers.ModelSerializer):
+    class Meta:
+        fields = [
+            'id',
+            'device',
+            'wifi_client',
```

Comment:
in order to make this API endpoint useful as the admin list of the wifi sessions, we must show the wifi client info here, I'd do that with a nested serializer which includes all the info of the wifi client here, eg:

```
{
  "id": "5db58413-548d-494b-a68a-a20ab63415be",
  "device": "be6bc1ba-4215-4512-8a1f-743b69fbfe4b",
  "client": {
      "mac_address": "b0:e1:7e:00:00:00",
      "vendor": "HUAWEI TECHNOLOGIES CO.,LTD",
      "ht": true,
      "vht": false,
      "wmm": true,
      "wds": false,
      "wps": false
  },
  "ssid": "openwisp-test",
  "interface_name": "wlan1",
  "start_time": "2022-05-30T18:54:07.225321+02:00",
  "stop_time": "2022-05-30T18:54:34.082815+02:00"
}
``` 

I would call it `client` instead of `wifi_client` for brevity and also because we know we're dealing with wifi sessions.

Make sure the query used in the view to fetch the wifi session info uses `select_related` to join wifi clients and wifi sessions to avoid generating too many queries.

For the sake of completeness I would add the following fields too:

- device_name
- organization_id
- organization_name

Still making sure all this info is fetched with `select_related`.